### PR TITLE
feat(quote): remove internal fields from schema

### LINKED ISSUE #2856: Update the `createQuote` mutation response type
### Context
In the `CreateQuote` method, we are returning a few ILP-specific fields: `maxPacketAmount` `minExchangeRate` `lowEstimatedExchangeRate` `highEstimatedExchangeRate`. These are used internally only, and unlikely to be useful to any callers of the Admin APIs, so they can be removed from the resulting schema. They become especially unnecessary with the introduction of local payments.

However, it _would_ be useful to start exposing `estimatedExchangeRate` in the response object, as it can give integrators an easy way to display quote details to their end users.

### PR DESCRIPTION
**Pull request by golobitch**
## Changes proposed in this pull request

* Remove internal fields from quote schema

## Context

fixes #2856 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass
- [x] Bruno collection updated

### CODE REVIEW

File: packages/backend/src/open_payments/quote/service.test.ts
Comment (BlairCurrey) [inline packages/backend/src/open_payments/quote/service.test.ts:1]: This file can stay like it was (that is, reverting these changes). The quote service may still (and at this point, always should) return the additional ilp fields.

**Review by BlairCurrey — COMMENTED** [review]
1 item to change https://github.com/interledger/rafiki/pull/2858/files#r1718632922, otherwise looks good

**Comment by golobitch** [comment]
> 1 item to change https://github.com/interledger/rafiki/pull/2858/files#r1718632922, otherwise looks good

Yes, my bad. I reverted that file.

**Review by BlairCurrey — APPROVED** [review]

File: bruno/collections/Rafiki/Rafiki Admin APIs/Cancel Outgoing Payment.bru [hunk 1]
```diff
@@ -22,11 +22,8 @@ body:graphql {
         quote {
           createdAt
           expiresAt
-          highEstimatedExchangeRate
           id
-          lowEstimatedExchangeRate
-          maxPacketAmount
-          minExchangeRate
+          estimatedExchangeRate
```
Comment (mkurapov) [inline bruno/collections/Rafiki/Rafiki Admin APIs/Cancel Outgoing Payment.bru:26]: we also need to update the `Examples` folder (2 `Create Quote` requests in there)

**Review by mkurapov — COMMENTED** [review]
Looks good, just need to edit some bruno requests

File: bruno/collections/Rafiki/Rafiki Admin APIs/Cancel Outgoing Payment.bru (hunk 1 above)
Comment (golobitch) [reply bruno/collections/Rafiki/Rafiki Admin APIs/Cancel Outgoing Payment.bru:26]: Nice catch. I removed everything from bruno dir.

**Review by BlairCurrey — APPROVED** [review]

**Review by mkurapov — APPROVED** [review]

**Pull request merged**